spec/venue_spec.rb

```diff
@@ -44,12 +44,39 @@
 
   describe '#yell_at_patrons' do
     it 'returns a list of uppercased names' do
-      skip
+       # skip
       venue = Venue.new('Bluebird', 4)
       venue.add_patron('Mike')
       venue.add_patron('Megan')
       venue.add_patron('Bob')
       expect(venue.yell_at_patrons).to eq ['MIKE', 'MEGAN', 'BOB']
     end
   end
+
+  describe '#over_capacity' do
+    it 'returns true if venue is over capacity' do
```

Comment:
Make sure you are paying close attention to the interaction pattern. Your `over_capacity` is missing something and it wanted you to test for both `true` and `false`

```
pry(main)> venue = Venue.new('Bluebird', 4)
=> #<Venue:0x00007fed7c11dbd8 @capacity=4, @name="Bluebird", @patrons=[]>

pry(main)> venue.add_patron('Mike')
pry(main)> venue.add_patron('Megan')
pry(main)> venue.add_patron('Bob')

pry(main)> venue.over_capacity?
=> false

pry(main)> venue.add_patron('James')
pry(main)> venue.add_patron('Cat')

pry(main)> venue.over_capacity?
=> true
```

which you can do in a single test